vcbuild.bat

```diff
@@ -201,6 +203,8 @@ if defined msi (
 call "%VS140COMNTOOLS%\..\..\vc\vcvarsall.bat"
 SET VCVARS_VER=140
 if not defined VCINSTALLDIR goto msbuild-not-found
+@rem Visual C++ Build Tools 2015 does not define VisualStudioVersion
```

Comment:
"Visual C++ Build Tools 2015" != "Build Tools 2015"... I'm sure VCBT doesn't set this.